llvm/test/CodeGen/Hexagon/iss127296.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: llc -mtriple=hexagon -O0 < %s | FileCheck %s
+; REQUIRES: asserts
+
+target datalayout = "e-m:e-p:32:32:32-a:0-n16:32-i64:64:64-i32:32:32-i16:16:16-i1:8:8-f32:32:32-f64:64:64-v32:32:32-v64:64:64-v512:512:512-v1024:1024:1024-v2048:2048:2048"
+target triple = "hexagon-unknown-linux4.19.0-musl"
+
+; CHECK:  r0 = #-1
+
+define fastcc void @os.linux.tls.initStatic() {
+	%1 = call { i32, i1 } @llvm.usub.with.overflow.i32(i32 0, i32 1)
+	br label %2
+
+	2:                                                ; preds = %0
+	%3 = extractvalue { i32, i1 } %1, 0
+	ret void
```

Comment:
Can you please replace the tabs here with spaces? Two spaces would be typical for LLVM.

Reply:
Fixed, thanks.